packages/@react-aria/combobox/src/useComboBox.ts

```diff
@@ -309,6 +309,11 @@ export function useComboBox<T>(props: AriaComboBoxProps<T>, state: ComboBoxState
       // This disable's the macOS Safari spell check auto corrections.
       spellCheck: 'false'
     }),
-    listBoxProps: mergeProps(menuProps, listBoxProps)
+    listBoxProps: mergeProps(menuProps, listBoxProps, {
+      autoFocus: state.focusStrategy,
+      shouldUseVirtualFocus: true,
```

Comment:
isn't this making too much of an assumption? and is it a breaking change now that it can't be spread onto any old html element like before?

Reply:
Yeah possibly. But in reality, you were meant to be passing it to `useListBox` before as well or it wouldn't have worked anyway. So I was hoping it would be ok...

Reply:
Yeah, I think this should be ok as well, combobox as a whole relies on virtual focus for the menu items so I don't think people would want to set shouldUseVirtualFocus to false.

+1 to the breaking change bit that @snowystinger mentioned though. Will we be making note of it somewhere when release notes go out?